modules/Emails/Email.php

```diff
@@ -4225,4 +4234,47 @@ protected function handleBodyInPlainTextFormat($mail)
         $mail->Body = $this->decodeDuringSend($mail->Body);
         $this->description = $mail->Body;
     }
+
+    /**
+     *
+     * @global array $sugar_config
+     * @global \LoggerManager $log
+     * @param string $emailField
+     * @return \EmailAddress
+     * @throws RuntimeException
+     * @throws InvalidArgumentException
+     */
+    public function fromSugarEmailAddressField($emailField)
+    {
```

Comment:
This function body still duplicated code. Exactly same method name and body lines in `include/SugarObjects/templates/basic/Basic.php` at line 142 and line 144 to 156.

Reply:
Sorry I didn't see that in the email.php. I have removed it